collect_app/src/main/java/org/odk/collect/android/activities/FormManagerList.java

```diff
@@ -265,12 +264,14 @@ public void syncComplete(String result) {
     public void deleteComplete(int deletedForms) {
         Log.i(t, "Delete forms complete");
         logger.logAction(this, "deleteComplete", Integer.toString(deletedForms));
-        if (deletedForms == getCheckedCount()) {
+        final int checkedCount = mBackgroundTasks.mDeleteFormsTask.getToDeleteCount();
```

Comment:
Same comment as above. By the time we're here, it's no longer the checked count even from the lists' perspective. It truly represents the count of stuff we previously meant to delete.